### ISSUE TITLE
[A11y] Improve accessibility for the Accordion block

### ISSUE THREAD

**Issue by jlemay-godaddy** [godaddy-wordpress/coblocks#1962]
**Accessibility problem:**
Keyboard navigation was couldn't open it after landing on it. When I expand the drop down menu, there is visual focus only if using the tab key to navigate through. If I use the arrow key it announces the menu item but does not have a visual focus indicator

Requested in #1927
